NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
```

Comment:
function add (x ,y){
 	var addition = x + y;
	if (typeof addition === "string"){
		alert("This function only accepts numbers");
	}
	else {return addition};
}

I solved it :) Thanks.